livre/eq/Channel.cpp

```diff
@@ -118,8 +147,9 @@ struct Channel::Impl
 public:
     explicit Impl( Channel* channel )
           : _channel( channel )
-          , _frameInfo( Frustum( Matrix4f(), Matrix4f()), INVALID_FRAME )
+          , _frameInfo( Frustum( Matrix4f(), Matrix4f()), INVALID_FRAME, INVALID_FRAME )
           , _progress( "Loading bricks", 0 )
+          , _frameCounter( INVALID_FRAME )
```

Comment:
I see the problem. I don't like this solution very much but I don't have a better suggestion except changing Equalizer to provide the same information in frameDraw, which we won't do.